tigerpath/majors_and_certificates/majors/PHI_2018.json

```
@@ -15,143 +15,508 @@
+                    "PHI 340",
+                    "PHI 490"
+                  ]
+                },
+                {
+                  "name": "History of Philosophy",
+                  "max_counted": 1,
+                  "min_needed": 2,
+                  "explanation": null,
+                  "course_list": [
+                    "PHI 200",
+                    "PHI 205",
+                    "PHI 300",
+                    "PHI 301",
+                    "PHI 302",
+                    "PHI 303",
+                    "PHI 304",
+                    "PHI 306",
+                    "PHI 332",
+                    "PHI 333",
+                    "PHI 335",
+                    "PHI 338"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Two philosophy courses unrestricted as to distribution area.",
+              "course_list": [
+                "PHI *"
+              ]
+            }
+          ]
         },
         {
           "name": "Philosophy of Science",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "Philosophy of Science. Senior concentrators doing their theses in philosophy of science have the option of substituting for the usual distribution requirements (two courses in each of three areas plus two unrestricted courses) the following: two upper-division (300 level or higher) courses in one relevant science (such as mathematics, computer science, physics, biology, psychology, economics); two philosophy courses in the logic and philosophy of science area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
-          "no_req": true
+          "explanation": "Senior concentrators doing their theses in philosophy of science have the option of substituting for the usual distribution requirements (two courses in each of three areas plus two unrestricted courses) the following: two upper-division (300 level or higher) courses in one relevant science (such as mathematics, computer science, physics, biology, psychology, economics); two philosophy courses in the logic and philosophy of science area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
+          "req_list": [
+            {
+              "name": "Science",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Two upper-division (300 level or higher) courses in one relevant science (such as mathematics, computer science, physics, biology, psychology, economics).",
+              "course_list": [
+                "COS 3*",
+                "COS 4*",
+                "COS 5*",
+                "ECO 3*",
+                "ECO 4*",
+                "ECO 5*",

```

EEB is also biology.
```suggestion
                "ECO 5*",
                "EEB 3*",
                "EEB 4*",
                "EEB 5*",
```